index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      @media only screen and (max-width: 1200px){
+    /*Tablets [601px -> 1200px]*/
+      }
+      @media only screen and (max-width: 600px){
+        /*Big smartphones [426px -> 600px]*/
+      }
+      @media only screen and (max-width: 425px){
+        /*Small smartphones [325px -> 425px]*/
+      }
+
+      .flex-container {
+      display: flex;
+    }
+
+    header {
+  position: fixed;
+  width: 100%;
+  top: 0;
+  left: 0;
+}
+
+    </style>
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <header id="header">
+
+      <nav id="nav-bar">
+        <a href="#form" class="nav-link">Form</a><br>
+        <a href="#video" class="nav-link">Video</a><br>
+        <a href="#header-img" class="nav-link">Image</a><br>
+      </nav>
+
+      <img id="header-img" src="https://deliverypharmacyke.com/wp-content/uploads/2021/12/HEAD-SHOULDERS-CLASSIC-CLEAN-ANTI-DANDRUFF-SHAMPOO-500ml.jpg"></img>
+      
+    </header>
+
+    
+    <h1>Shampoo</h1>
+    
+
```

Comment:
For linter checking it would be better to remove all empty lines otherwise the linter will raise error.

Reply:
- Great comment 👍🏼 that includes a polite suggestion and why requesting the changes. keep up the good work 💯